### DISCUSSION TITLE
Pregunta TPA Grupo 1. Creacion de formularios

### DISCUSSION THREAD

**Discussion by MatiasCornalo** [dds-jv/foro#415] [TPA - 2024] [answered]
Buenas tardes, tenemos la siguiente duda:
Los formularios se tienen  que crear cada vez que el colaborador (humano o  jurídico) desea colaborar o se crean una única vez cuando se da de alta / crea el colaborador.

**Comment by flbulgarelli** [comment, marked as answer]
:wave: ¡Hola, @MatiasCornalo!

Los formularios no son (o, al menos, no necesariamente)  entidades del sistema que deban ser "creadas" como tales. Los mismos son tan sólo un medio para el alta, para quien colabore pueda ingresar sus datos. 

Ahora bien, si la pregunta está orientada a si los datos de las partes colaboradoras deben ser cargados cada vez que se realiza una colaboración, o si en cambio se debe dar de alta una única vez, entonces es lo segundo. ¿Por qué? Porque salvo que el anonimato de quien colabora sea un requisito (que en esta entrega no lo es), solicitar la misma información una y otra vez sería engorroso y propenso a errores sin reportar ningún beneficio. 

Decime si queda claro. 

¡Saludos!

**Comment by MatiasCornalo** [comment]
Quedo clarísimo. Gracias profe!
